dialout/dialout_client_cli/dialout_client_cli.go

```diff
@@ -19,13 +20,15 @@ var (
 		RetryInterval:  30 * time.Second,
 		Encoding:       gpb.Encoding_JSON_IETF,
 		Unidirectional: true,
-		TLS:            &tls.Config{},
 	}
+	tlsCfg     = tls.Config{}
+	tlsDisable bool
 )
 
 func init() {
-	flag.StringVar(&clientCfg.TLS.ServerName, "server_name", "", "When set, use this hostname to verify server certificate during TLS handshake.")
-	flag.BoolVar(&clientCfg.TLS.InsecureSkipVerify, "insecure", false, "When set, client will not verify the server certificate during TLS handshake.")
+	flag.StringVar(&tlsCfg.ServerName, "server_name", "", "When set, use this hostname to verify server certificate during TLS handshake.")
+	flag.BoolVar(&tlsCfg.InsecureSkipVerify, "skip_verify", false, "When set, client will not verify the server certificate during TLS handshake.")
```

Comment:
you should keep the old flag name, otherwise it can cause other people's test regression test fail. Instead, you should rename you new flag

Reply:
like disable_tls

Reply:
Change has been made. Insecure flag has been reverted back to corresponding with tlsCfg.InsecureSkipVerify and created a new flag, "tls_disable" for the new tlsDisable option.